File: 02-is-at-least-one-pair-of-equal-numbers/main.py
#!/usr/bin/python3

def main():
    a = float(input("Введіть число a: "))
    b = float(input("Введіть число b: "))
    c = float(input("Введіть число c: "))

    # Чи є хоча б одна пара чисел рівних між собою
    if a == b:
        is_equal = True
    elif a == c:
        is_equal = True
    elif b == c:
        is_equal = True
    else:
        is_equal = False

    result = "є хоча б одна пара" if is_equal else "немає хоча б однієї пари"
    print("Серед цих чисел", result, "рівних між собою")


if __name__ == "__main__":
    main()

File: 02-is-at-least-one-pair-of-equal-numbers/main.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    float a, b, c;  // Вхідні числа a, b, c

    cout << "Введіть число a: "; cin >> a;
    cout << "Введіть число b: "; cin >> b;
    cout << "Введіть число c: "; cin >> c;

    // Чи є хоча б одна пара чисел рівних між собою
    bool is_equal = (a == b) || (a == c) || (b == c);

    string result = (is_equal) ? "є хоча б одна пара" : "немає хоча б однієї пари";
    cout << "Серед цих чисел " << result << " рівних між собою\n";

    return 0;
}